src/utils/exportUtils/HtmlExport.ts

```diff
@@ -0,0 +1,459 @@
+
+import streamSaver from "streamsaver";
```

Comment:
please write all new code as Typescript

Reply:
This was changed to TypeScript in the later commit